src/firefox/remote.js

```diff
@@ -86,27 +86,43 @@ export class RemoteFirefox {
     });
   }
 
+  getAddonsActor(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      // getRoot should work since Firefox 55 (bug 1352157).
+      this.client.request('getRoot', (ignoredError, actors) => {
+        if (actors && actors.addonsActor) {
+          resolve(actors.addonsActor);
+          return;
+        }
+        log.debug('getRoot failed, falling back to listTabs');
```

Comment:
Nit: would you mind to include the `ignoredError` in this debug message?
just to make investigate issues easier in case the `getRoot` request failed for unrelated reasons to Firefox version being older than 55.